Formula/s/superfile.rb

```diff
@@ -0,0 +1,19 @@
+class Superfile < Formula
+  desc "Modern and pretty fancy file manager for the terminal"
+  homepage "https://github.com/MHNightCat/superfile"
+  url "https://github.com/MHNightCat/superfile/archive/refs/tags/v1.1.0.tar.gz"
+  sha256 "703ab37f1bb994e7decc0b48200dd82b24e58ddff8d3204d9cb53fed67ef0f4a"
+  license "MIT"
+  depends_on "go" => :build
+
+  def install
+    cd "src" do
+      system "go", "build", *std_go_args(ldflags: "-s -w", output: bin/"spf")
+    end
```

Comment:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w", output: bin/"spf"), ".../src"
```

Reply:
This seems to make it can't to install :(

Reply:
the issue is go.mod is not in the root dir.